test/common/README.md

```diff
@@ -8,6 +8,7 @@ This directory contains modules used to test the Node.js implementation.
 * [Common module API](#common-module-api)
 * [Countdown module](#countdown-module)
 * [DNS module](#dns-module)
+* [Duplex pair helper](#duplex-pair-module)
```

Comment:
Nit: should this be `Duplex pair Module` to match heading below?

Reply:
If you want this changed for consistency, I’d go with `Duplex pair helper` everywhere. Does that seem okay?

Reply:
That seems good 👍

Reply:
ok, done!